Automattic/wp-calypso
# Checkout: Always use foldable card for non-critical Terms of Service

Checkout has a very large number of "terms of service" (TOS) notices which are displayed at the bottom of the checkout form before the payment button. These notices can in some cases take a up large amount of vertical space and make the checkout experience feel cluttered. To help improve this situation, an AB test was run to determine how useful it would be to hide the non-essential TOS notices behind a foldable card that says "Read more". This was only for TOS notices which were not legally required to be visible without a click. The test showed no danger with this approach and so it was launched to most checkout sections in https://github.com/Automattic/wp-calypso/pull/85445. Additional sections were launched in https://github.com/Automattic/wp-calypso/pull/86215.

As we near the potential launch of the full checkout v2 design, I believe that we should launch the foldable card for all instances of checkout.

## Proposed Changes

This PR activates the foldable "Read more" card for all versions of checkout. Specifically, this launches the new UX for Jetpack checkout and Akismet checkout which were the only two remaining holdouts.

Note that in most cases, Jetpack and Akismet checkout currently do not actually have anything in the "Read more" card and so they will not show it. The main change therefore will be the small UX differences like adding the refund icon and removing the "i" icon. See the screenshots below.

As a side effect, this fixes https://github.com/Automattic/wp-calypso/issues/87368.

## Screenshots

What | Before             |  After
:-----:|:-------------------------:|:-------------------------:
Akismet | <img width="610" alt="before-87554-akismet" src="https://github.com/Automattic/wp-calypso/assets/2036909/b5962b76-c629-4c6f-99d4-7905e5ba4bf8"> | <img width="605" alt="after-87554-akismet" src="https://github.com/Automattic/wp-calypso/assets/2036909/d284dc95-ca46-455a-8f02-21b39b892d46">
Jetpack plan | <img width="608" alt="before-87554-jetpack-plan" src="https://github.com/Automattic/wp-calypso/assets/2036909/2ef2c6f3-d9b1-4419-91ee-b60dad19bdd8"> | <img width="612" alt="after-87554-jetpack-plan" src="https://github.com/Automattic/wp-calypso/assets/2036909/9dd662e2-efd7-42a1-8383-c4a957be6a44">
Jetpack one-time | <img width="604" alt="before-87554-jetpack-one-time" src="https://github.com/Automattic/wp-calypso/assets/2036909/4678fba8-0ff4-4221-a1f4-80a3a7127b26"> | <img width="606" alt="after-87554-jetpack-one-time" src="https://github.com/Automattic/wp-calypso/assets/2036909/dded8913-fd3f-4c4b-9dea-84dc29430630">


## Testing Instructions

- Visit checkout for a Jetpack site (eg: `/checkout/jetpack/jetpack_complete`) or for an Akismet product (eg: `/checkout/akismet/ak_plus_yearly_2`).
- Verify that the UX of the TOS at the bottom of the page looks reasonable.
